src/lib/useLocalStorageState.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react';
+
+/**
+ * useLocalStorageState is useState but it persists to localStorage
+ *
+ * inspired by:
+ * https://github.com/streamich/react-use/blob/master/src/useLocalStorage.ts
+ * https://usehooks.com/useLocalStorage/
+ */
+export default function useLocalStorageState(key, initialValue) {
```

Comment:
Nice! Do you think it's worth updating the Disclaimer / hasDisclaimed to use this also, instead of dumb cookie usage?

Reply:
yes, will do that